### ISSUE TITLE
java.lang.RuntimeException: java.time.format.DateTimeParseException: Text '0' could not be parsed at index 0

### ISSUE THREAD

**Issue by mkhokh-33** [airbytehq/airbyte#15840]
created issue from @danieldiamond  comment ralated to RP - https://github.com/airbytehq/airbyte/pull/15504
A couple of people running into issues now
java.lang.RuntimeException: java.lang.RuntimeException: java.time.format.DateTimeParseException: Text '0' could not be parsed at index 0

e.g.

- myself with source MySQL from 0.6.2 -> 0.6.4 (I'm using CDC). confirmed that syncs work again when reverting back to 0.6.2
- someone in disourse [Potgres sync fail after update 0.4.0 -> 1.0.2](https://discuss.airbyte.io/t/potgres-sync-fail-after-update-0-4-0-1-0-2/2270)
- and [someone else in slack](https://airbytehq.slack.com/archives/C038KM9NF7Z/p1660926801819989)

**Comment by subodh1810** [comment]
This seems relevant 
```
The thing is it happens only in incremental mode (I don’t get the issue in full refresh mode).
One column has a default value set as CURRENT_TIMESTAMP(0) and this is slightly different from other tables that have a default value set as CURRENT_TIMESTAMP
Could it be the root cause ?
```

**Comment by grishick** [comment]
@mkhokh-33  This is likely to cause a P0 outage, so please take this into implementation ASAP prioritizing above the other issue that you have in review currently.

**Comment by ryankfu** [comment]
@mkhokh-33 it seems the likely culprit is actually related to this PR: https://github.com/airbytehq/airbyte/pull/15504/files since this was not added to the change log but it touched `MySQL`. There may be another PR that touched `Postgres` but this is where the team seems to think the issue arose

**Comment by subodh1810** [comment]
@danieldiamond 
I am not able to reproduce this for mysql or postgres. I tried to sync data from a mysql table with the following structure in CDC mode and it worked perfectly fine
```
create table test(id int primary key, updated_at timestamp(0) not null default CURRENT_TIMESTAMP(0), updated_at_2 timestamp(1) not null default CURRENT_TIMESTAMP(1), updated_at_3 timestamp(2) not null default CURRENT_TIMESTAMP(2), updated_at_4 timestamp(3) not null default CURRENT_TIMESTAMP(3), updated_at_5 timestamp(4) not null default CURRENT_TIMESTAMP(4), updated_at_6 timestamp(5) not null default CURRENT_TIMESTAMP(5));
```

can you share the schema of the table for which you faced the error and logs from the sync

**Comment by mackro-rocky** [comment]
We're seeing the same issue on postgres with 1.0.2 connector.

Logs attached
[logs-56.txt](https://github.com/airbytehq/airbyte/files/9397136/logs-56.txt)

**Comment by subodh1810** [comment]
hey @mkhokh-33 FYI, am already working on fixing this.

**Comment by subodh1810** [comment]
Postgres error should be fixed by this PR https://github.com/airbytehq/airbyte/pull/15877

**Comment by subodh1810** [comment]
MySQL error should be fixed by this PR https://github.com/airbytehq/airbyte/pull/15917

**Comment by subodh1810** [comment]
This has been fixed for both postgres (in version 1.0.4) and MySQL (in version 0.6.5), closing the issue
